nss-day-cohort-51/nss-day-cohort-51.github.io
# mw-feature-ERD-4 Created directory under public called /images; Expor…

#4 
# Description - Create Cohort Website ERD
Created directory under public called /images; 
Exported the ERD from dbdiagram as a PNG file; 
uploaded the file to /images directory; 
added subheading and image code to the end of the README and embedded the ERD image file; 
ran preview of the README.md file in VSCode to check that image shows correctly.

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions
- Pull down branch and run VSCode.
- Verify file in **/images** directory named **ERD-C51-Cohort-Website.png** is an image of the ERD.
- Right-click README.md and click **Open Preview**. The ERD image should be visible at the bottom of README.md.

ADDITION:

# Checklist:

- (N/A) I have commented my code, particularly in hard-to-understand areas
